app/controllers/lessons_controller.rb

```diff
@@ -0,0 +1,64 @@
+class LessonsController < ApplicationController
+  PER_PAGE = 6
+
+  def index
+    return desc_order_lessons if params[:order] == 'desc'
```

Comment:
да

On Wed, Mar 16, 2016 at 7:23 PM, Leonidas777 <notifications@github.com>
wrote:

> In app/controllers/lessons_controller.rb
> <https://github.com/Leonidas777/course-app/pull/10#discussion_r56353618>:
>
> > @@ -0,0 +1,64 @@
> > +class LessonsController < ApplicationController
> > +  PER_PAGE = 6
> > +
> > +  def index
> > +    return desc_order_lessons if params[:order] == 'desc'
>
> Хочу уточнить. Имеется в виду сортировка по позиции, которую автор
> устанавливает при создании урока?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Leonidas777/course-app/pull/10/files/fb3e8d95d89ffc56fa0728eb40f3312c5dfb87be#r56353618>
>



-- 
[image: jetruby] <http://jetruby.com/>
[image: .]
OLEG ZAPOROZHCHENKORUBY ON RAILS DEVELOPER

[image: .]
P.  415-670-9085M.  medium.com/@jetruby